src/views/Items/Item.js

```diff
@@ -0,0 +1,47 @@
+import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
+
+export default class Item extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    description: (props, propName, componentName) => {
+      if (props.children && props.description) {
```

Comment:
prop validation that spits out an error if both `children` and `description` exists.